FantasyUniverse/Views/SignUpView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SignUpView.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 14.05.2022.
+//
+
+import SwiftUI
+
+struct SignUpView: View {
+    @ObservedObject var viewModel = SignUpViewModel()
+    
+    @State private var firstNameFieldValue = ""
+    @State private var lastNameFieldValue = ""
+    @State private var loginFieldValue = ""
+    @State private var passwordFieldValue = ""
```

Comment:
If you're going for MVVM pattern then these should be published properties in the view model and check credentials should react on the changes in view model so that logic is not leaking into views.